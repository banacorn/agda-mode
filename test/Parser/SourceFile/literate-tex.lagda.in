Ignored by Agda.

\begin{code}[ignored by Agda]
module Whatever where
-- Agda code goes here

a = ?

b = {! !}


\end{code}