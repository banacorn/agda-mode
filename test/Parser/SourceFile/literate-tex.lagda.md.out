Hole [0] (190, 196) => (190, 196) "{!  !}"